old-pages/contents/about.ts

```diff
@@ -20,88 +17,128 @@ interface AboutMainBody {
   about_main_content: string;
 }
 
+interface Profile {
+  profile_1: string;
+  profile_2: string;
+  profile_3: string;
+  profile_4?: string;
+  profile_5?: string;
+}
+
 interface AboutUsTeam {
   id: string;
+  company: string;
   image: StaticImageData;
-  imageProfile: StaticImageData
+  imageProfile: StaticImageData;
   fullname: string;
   post: string;
-  slug: string
   personality?: string;
   view_profile: string;
-  profile_1: string;
-  profile_2: string;
-  profile_3: string;
-  profile_4?: string;
-  profile_5?: string;
+  profile: [...Array<Profile>];
 }
 
 export const about_hero: AboutHero[] = [
   {
-    id: 'about_hero',
+    id: "about_hero",
     hero_content:
-      'We are your trusted guide for an unforgettable Hajj experience. We craft transformative pilgrimages that nurture spiritual growth and ignite intellectual curiosity.',
+      "We are your trusted guide for an unforgettable Hajj experience. We craft transformative pilgrimages that nurture spiritual growth and ignite intellectual curiosity.",
   },
 ];
 
 export const about_main_body: AboutMainBody[] = [
   {
-    id: 'about_main',
+    id: "about_main",
     about_main_content:
-      'At Almwaqeet, we make the sacred journeys of Hajj and Umrah accessible, comfortable, and fulfilling. With a focus on supporting pilgrims, we offer flexible payment plans, expert-guided tours, and seamless travel arrangements for every budget. We handle all the details—booking, transportation, and accommodations—so you can focus on your spiritual journey with complete peace of mind.',
+      "Al-Mawaqeet Travels and Tours stands as a premier Islamic travel agency, committed to delivering unparalleled pilgrimage and educational experiences to travelers in search of spiritual enlightenment and academic exploration. Our dedicated team, comprising erudite Islamic scholars and core professionals, is devoted to ensuring a seamless journey for each traveler, guaranteeing safety, satisfaction, and an experience that will be etched in their memories.",
   },
 ];
 
 export const about_us_team: AboutUsTeam[] = [
   {
-    id: 'img1',
-    slug: 'Shaykh-Abdullah',
+    id: "Baba-n-Madeenah",
```

Comment:
slugs are usually all lower case separated with hyphens, it wouldnt make sense to have upper case characters in our url. please change this to lower case